.ci/build-n-run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+function build_example()
+{   
+    make -C examples || exit 1
+}
+
+function list_mod()
+{
+    #following list will contain all filename which doesn't appear in non-working without path and file extension
```

Comment:
Check the grammar. Should be
> all file names which are not specified in file non-working.

In UNIX-like systems, there is no distinct concept about "file extensions," and you can find various files starting with dot (`.`) such as `.bashrc` and `.config`.